openassessment/xblock/static/js/src/studio/oa_container_item.js

```diff
@@ -75,9 +75,17 @@ OpenAssessment.ItemUtilities = {
  Returns:
  OpenAssessment.Prompt
  **/
-OpenAssessment.Prompt = function(element, notifier) {
+OpenAssessment.Prompt = function(element, notifier, kwargs) {
```

Comment:
I'll add comment but i disagree about `kwargs`:
As I may see from the code base - all elements are implemented as the `Container` items: https://github.com/edx/edx-ora2/blob/master/openassessment/xblock/static/js/src/studio/oa_container.js#L75
It is `Prompt`, `RubricOption`, `RubricCriterion` and etc. So i can't just pass some additional parameter to `Prompt` constructor. Of course it will not do anything wrong if I'll pass it to all `ContainerItem`-s but as for my opinion it isn't good solution. Also I may add `if` statement but as for my opinion current solution with `kwargs` is universal and more elegant